panther_analysis_tool/main.py

```diff
@@ -463,15 +464,11 @@ def update_custom_schemas(args: argparse.Namespace) -> Tuple[int, str]:
     Returns:
         A tuple of return code and a placeholder string.
     """
-    if args.aws_profile is not None:
-        logging.info("Using AWS profile: %s", args.aws_profile)
-        set_env("AWS_PROFILE", args.aws_profile)
-
     normalized_path = user_defined.normalize_path(args.path)
     if not normalized_path:
         return 1, f"path not found: {args.path}"
 
-    uploader = user_defined.Uploader(normalized_path)
+    uploader = user_defined.Uploader(normalized_path, args)
```

Comment:
Since most of the other arguments are specific to main (zip/test/output directory) - wonder if we could simply pass in the aws profile here rather than the entire arg list? Or do you envision other arguments may be pertinent to the user schemas at some point in the future?

Reply:
Alternatively, you could also (optionally so that tests are unaffected) inject the client instance directly.